coding-challenges.js

```diff
@@ -36,6 +70,17 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
+const extra = (arr) => {
+    arr.sort((a,b) => a-b)
+    let front = arr.shift()
+    let end = arr.pop()
+    let finalArr = []
+    finalArr.push(front)
+    finalArr.push(end)
+    return finalArr
+}
```

Comment:
great! you can also make this even shorter by writing `finalArr.push(arr.shift(), arr.pop())` if you'd like! :)